app/src/main/res/values-fr/strings.xml

```diff
@@ -180,7 +180,7 @@
     <string name="uploadSuccessLinkTitle">Ton transfert est prêt !</string>
     <string name="uploadSuccessQrTitle">Ton QR code est prêt !</string>
     <string name="urlAbout">https://www.infomaniak.com/fr/a-propos</string>
-    <string name="urlUserReportAndroid">https://feedback.userreport.com/1c462a20-7559-415e-a6e0-4b624dc38877/#ideas/popular</string>
+    <string name="urlUserReport">https://feedback.userreport.com/1c462a20-7559-415e-a6e0-4b624dc38877/#ideas/popular</string>
```

Comment:
Same than the spanish one. Is it on purpose ?

Reply:
Looks like it's the 🇪🇸 one that is wrong.
The link is the one for the 🇫🇷 language.

Reply:
Nice catch, I updated the strings it's fixed